lib/tapioca/config_builder.rb

```diff
@@ -10,6 +10,9 @@ class << self
 
       sig { params(command: Symbol, options: T::Hash[String, T.untyped]).returns(Config) }
       def from_options(command, options)
+        puts(<<~MSG) if options.include?("generate_command")
+          DEPRECATION: The `-c` and `--cmd` flags will be removed in a future release.
+        MSG
         Config.from_hash(
           merge_options(default_options(command), config_options, options)
         )
```

Comment:
Small improvement might be to make sure that we don't get that option set by any config file either:
```suggestion
        merged_options = merge_options(default_options(command), config_options, options)

        puts(<<~MSG) if merged_options.include?("generate_command")
          DEPRECATION: The `-c` and `--cmd` flags will be removed in a future release.
        MSG
        
        Config.from_hash(merged_options)
```